core/cautils/customerloader.go

```diff
@@ -63,6 +67,44 @@ func (co *ConfigObj) Config() []byte {
 	return []byte{}
 }
 
+func (co *ConfigObj) UpdateEmptyFields(inCO *ConfigObj) error {
+	if inCO.AccountID != "" {
+		co.AccountID = inCO.AccountID
+	}
+	if inCO.ClientID != "" {
+		co.ClientID = inCO.ClientID
+	}
+	if inCO.CloudAPIURL != "" {
+		co.CloudAPIURL = inCO.CloudAPIURL
+	}
+	if inCO.CloudAuthURL != "" {
+		co.CloudAuthURL = inCO.CloudAuthURL
+	}
+	if inCO.CloudReportURL != "" {
+		co.CloudReportURL = inCO.CloudReportURL
+	}
+	if inCO.CloudUIURL != "" {
+		co.CloudUIURL = inCO.CloudUIURL
+	}
+	if inCO.ClusterName != "" {
+		co.ClusterName = inCO.ClusterName
+	}
+	if inCO.CustomerAdminEMail != "" {
+		co.CustomerAdminEMail = inCO.CustomerAdminEMail
+	}
+	if inCO.SecretKey != "" {
+		co.SecretKey = inCO.SecretKey
+	}
+	if inCO.Token != "" {
+		co.Token = inCO.Token
+	}
+	if inCO.CustomerGUID != "" {
```

Comment:
CustomerGUID is deprecated.
You can fully ignore it.
Also, you can remove it from the config struct definition